core/src/main/scala/org/apache/spark/storage/ShuffleBlockFetcherIterator.scala

```diff
@@ -29,7 +29,6 @@ import scala.collection.mutable.{ArrayBuffer, HashMap, HashSet, Queue}
 import scala.util.{Failure, Success}
 
 import io.netty.util.internal.OutOfDirectMemoryError
-import org.apache.commons.io.IOUtils
```

Comment:
BTW, do you know why it's deprecated?

Reply:
```
@deprecated As of 2.6 deprecated without replacement. Please use the try-with-resources statement or handle
suppressed exceptions manually.
```

All our original logic is suppressing `IOException`, so I think we can reuse `JavaUtils. closeQuietly` directly If all modules can access it.